app/lib/csv/buyers_exporter.rb

```diff
@@ -24,10 +25,10 @@ def values_for_account(account)
       account.bought_account_plan.try!(:name),
       account.created_at.to_s(:db),
       account.bought_cinstances.count,
-      account.first_admin.try!(:display_name),
-      account.first_admin.try!(:email),
+      account.decorate.admin_user_display_name,
+      user&.email,
       account.extra_fields.to_json,
-      account.first_admin.try!(:extra_fields).to_json
+      (user&.extra_fields || {}).to_json
```

Comment:
I think this is not the same as before.

_Before:_ if there was no `account.first_admin`
```ruby
account.first_admin.try!(:extra_fields).to_json
=> "null"
```

_Now:_ if there's no `user`
```ruby
(user&.extra_fields || {}).to_json
=> "{}"
```

Reply:
Right 👍 And I will also add a test for that 😄